src/cpu/aarch64/jit_primitive_conf.hpp

```diff
@@ -0,0 +1,727 @@
+    int typesize_acc;
+    /* 4fma */
+    bool transpose_src;
+    int tr_is;
+    int nthr, nthr_mb, nthr_g, nthr_oc_b, nthr_ic_b;
+    int is_oc_scale;
+    data_type_t bia_dt;
+    data_type_t dst_dt;
+    bool signed_input;
+    float wei_adj_scale;
+    // zero-point compensation
+    bool src_zero_point;
+    bool dst_zero_point;
+    bool zp_src_is_common; // common, otherwise (TODO) per-channel
+
+    cpu_isa_t isa;
+    bool uses_permw_transposition;
+};
+
+struct jit_1x1_conv_call_s {
+    const void *bcast_data;
+    const void *load_data;
+    const void *output_data;
+    const void *bias_data; // used in forward and backward_weights only
+    const void *acc_s32;
+    const void *scales;
+    const void *compensation;
+    const void *store_buffer;
+    const int32_t *zp_compensation;
+    const int32_t *src_zero_point;
+    const int32_t *dst_zero_point;
+
+    // ptr to table of void * elements that are pointers to
+    // post_op binary src1 tensors
+    const void *post_ops_binary_rhs_arg_vec;
+    // logical (# of elems) offset to the processed output channel
+    // (for broadcasting [1,OC,1,1])
+    size_t oc_l_off;
+    const void *dst_orig; // pointer to dst memory (not offseted)
+
+    size_t load_dim;
+    size_t bcast_dim;
+    size_t reduce_dim;
+
+    size_t output_stride; // used in backward_weights only
+
+    size_t first_last_flag;
+};
+
+struct jit_pool_conf_t {
+    int ndims;
+    int mb, c, c_without_padding;
+    int id, ih, iw, od, oh, ow;
+    int stride_d, stride_h, stride_w;
+    int kd, kh, kw;
+    int f_pad, t_pad, l_pad;
+    alg_kind_t alg;
+    bool is_training;
+    bool pad_w_is_null;
+    bool is_backward;
+    bool simple_alg;
+    bool is_c_padded;
+    data_type_t ind_dt;
+
+    int c_block, c_tail, nb_c;
+    int ur_bc, ur_bc_tail;
+    int ur_c, ur_c_tail;
+    int ur;
+    size_t tail[4];
+    bool safe_c_tail;
+    data_type_t src_dt;
+    data_type_t dst_dt;
+
+    int dt_size;
+    bool is_bf16;
+    jit_memory_tag_kind_t tag_kind;
+    bool is_plain() const {
+        return (tag_kind == jit_memory_tag_kind_t::ncsp
+                || tag_kind == jit_memory_tag_kind_t::nspc);
+    }
+
+    cpu_isa_t isa;
+    post_ops_t post_ops;
+    bool with_postops;
+    bool with_eltwise;
+    bool with_binary;
+};
+
+struct jit_pool_call_s {
+    const void *src;
+    const void *dst;
+    const void *indices;
+    const void *src_prf;
+    const void *dst_prf;
+    const void *indices_prf;
+    const void *post_ops_binary_rhs_arg_vec;
+    size_t c_elem_off;
+    size_t zero_ih;
+    size_t zero_id;
+    const void *zero_ptr;
+    size_t kd_padding;
+    size_t kh_padding;
+    size_t kh_padding_shift;
+    size_t kd_padding_shift;
+    size_t kw_padding;
+    const void *init_value;
+    float ker_area_h;
+    size_t ur_bc; // contains number of channel blocks to processing
+    size_t b_c; // contains number of channel blocks already processed
+};
+
+struct jit_resampling_conf_t {
```

Comment:
[this and similarly included other `struct`]

As I understand, the commits support the addition of `int8` convolution so I was wondering if we need to get rid of  `jit_resampling_conf_t` , `jit_pool_call_s`, etc.

Reply:
+1
I'm observing a lot of x64 specific fields that aren't used in the aarch64 implementation. I suggest removing everything that's not required...

Reply:
Thx! I'll delete it.